Octokit.Tests.Integration/Clients/MigrationsClientTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Threading;
+using System.Threading.Tasks;
+using Octokit;
+using Octokit.Tests.Integration;
+using Octokit.Tests.Integration.Helpers;
+using Xunit;
+
+public class MigrationsClientTests : IDisposable
+{
+    private readonly IGitHubClient _gitHub;
+    private List<RepositoryContext> _repos;
+    private Migration _migrationContext;
+    private string _orgName;
+    private bool isExported = false;
+
+    public MigrationsClientTests()
+    {
+        _gitHub = Helper.GetAuthenticatedClient();
+        _orgName = Helper.Organization;
+        _repos = new List<RepositoryContext>();
+
+        CreateTheWorld().Wait();
+        StartNewMigration().Wait();
+    }
+
+    private async Task CreateTheWorld()
+    {
+        _repos.Add(await _gitHub.CreateRepositoryContext(_orgName, new NewRepository("migrationtest-repo1")
+        {
+            GitignoreTemplate = "VisualStudio", LicenseTemplate = "mit"
+        }));
+        _repos.Add(await _gitHub.CreateRepositoryContext(_orgName, new NewRepository("migrationtest-repo2")
+        {
+            GitignoreTemplate = "VisualStudio",
+            LicenseTemplate = "mit"
```

Comment:
the first repo creation has the GitIgnoreTemplate, LicenseTemplate on the same line, but the next 2 have these on separate lines.  would be nice to :lipstick: these to be consistent with eachother